scripts/extract_translation_templates.py

```diff
@@ -10,125 +10,136 @@
-        source_file = f"{source_dir}/{filename}"
-        target_file = f"{target_dir}/{filename}"
+    contents_match = source_contents == target_contents
 
-        source_contents = get_template_content(source_file, ignore_context=True)
-        target_contents = get_template_content(target_file, ignore_context=True)
+    if not contents_match:
+        diff_results = difflib.unified_diff(
+            source_contents,
+            target_contents,
+            fromfile=source_file,
+            tofile=target_file,
+        )
+        logger.info("".join(list(diff_results)))
 
-        contents_match = source_contents == target_contents
+    print_filename_results(f"{source_file}", contents_match)
 
-        if not contents_match:
-            diff_results = difflib.unified_diff(
-                source_contents,
-                target_contents,
-                fromfile=source_file,
-                tofile=target_file,
-            )
-            logger.info("".join(list(diff_results)))
+    return contents_match
 
-        print_filename_results(f"{source_file}", contents_match)
 
-        return contents_match
+def build_schema_templates(output_dir):
 
-    def build_schema_templates(output_dir):
+    for schema_name in SCHEMAS_TO_EXTRACT:
+        template_file = f"{schema_name}.pot"
+        schema_file = f"{schema_name}.json"
 
-        for schema_name in SCHEMAS_TO_EXTRACT:
-            template_file = f"{schema_name}.pot"
-            schema_file = f"{schema_name}.json"
+        logger.info("Building %s/%s", output_dir, template_file)
 
-            logger.info("Building %s/%s", output_dir, template_file)
+        handle_extract_template(f"schemas/en/{schema_file}", output_dir)
 
-            handle_extract_template(f"schemas/en/{schema_file}", output_dir)
+        logger.info("Built %s/%s", output_dir, template_file)
 
-            logger.info("Built %s/%s", output_dir, template_file)
 
-    def check_schema_templates(source_dir, target_dir):
-        return all(
-            compare_files(source_dir, target_dir, f"{schema_name}.pot")
-            for schema_name in SCHEMAS_TO_EXTRACT
-        )
+def check_schema_templates(source_dir, target_dir):
+    return all(
+        compare_files(source_dir, target_dir, f"{schema_name}.pot")
+        for schema_name in SCHEMAS_TO_EXTRACT
+    )
 
-    if __name__ == "__main__":
-        parser = argparse.ArgumentParser(
-            description="Extract translation templates from runner"
-        )
-        parser.add_argument(
-            "--test",
-            help="Test the templates without making changes",
-            action="store_true",
-        )
 
-        args = parser.parse_args()
+if __name__ == "__main__":
 
-        if args.test:
-            with tempfile.TemporaryDirectory(dir="/tmp") as temp_dir:
-                build_schema_templates(temp_dir)
+    page = requests.get("https://github.com/ONSdigital/eq-translations/commit/master/")
```

Comment:
We should use the GitHub API:
```python
response = requests.get("https://api.github.com/repos/ONSdigital/eq-translations/commits/master").json()
commit_sha = response.get('sha')
```
Then check `commit_sha == sha_from_pipfile`

Reply:
Done! Thanks!